<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerDialog</class>
 <widget class="QDialog" name="registerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>33</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kullanıcı Adı</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reg_usernameBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parola</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reg_passwordBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Gönder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_exit">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>140</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Statics/add-user.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
